### PR TITLE
Feature/breadcrumb

### PR DESCRIPTION
**Pull request by masagatech**
### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing guide](https://github.com/svelteuidev/svelteui/blob/main/CONTRIBUTING.md)
- [ ] Prefix your PR title with `[@svelteui/core]`, `[@svelteui/actions]`, `[@svelteui/motion]`, `[@svelteui/core]`, `[core]`, or `[docs]`.
- [ ] This message body should clearly illustrate what problems it solves.

### Tests

- [ ] Run the tests with `npm test` and lint the project with `yarn lint` or just run `yarn repo:prepush` and check to see if it's passing.

### CODE REVIEW

File: packages/svelteui-core/src/components/Breadcrumb/Breadcrumb.styles.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	lg: 12,
+	xl: 16
+};
+
+export default createStyles(
+	(theme, { color, size }: BreadcrumbStylesParams) => ({
+		root: {
+			// [`${theme.dark} &`]: {
+			// 	backgroundColor: theme.fn.themeColor('dark', 4)
+			// },
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Breadcrumb/Breadcrumb.styles.ts:23]: dead code?

File: packages/svelteui-core/src/components/Breadcrumb/Breadcrumb.svelte [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		color: $$Props['color'] = 'blue',
+		size: $$Props['size'] = 'md',
+		separetor: $$Props['separetor'] = '/';
+	export { className as class };
+
+	// initialize a 'reactive context' which is basically
+	// a store inside the context, so that all children
+	// components can react to changes made in props
+	const contextStore: BreadcrumbContext = writable({
+		separetor: separetor,
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Breadcrumb/Breadcrumb.svelte:28]: typo: separator

File: packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	md: 16,
+	lg: 18,
+	xl: 20
+};
+
+export default createStyles(
+	(theme, { color, size }: BreadcrumbItemStylesParams) => ({
+		root: {
+			[`${theme.dark} &`]: {
+				backgroundColor: theme.fn.themeColor('dark', 4)
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts:30]: Hum, I don't think it should contain the change in background color. The only thing that should change with the change in theme is the color of the separator

File: packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+				display: 'flex',
+				gap: size,
+				alignItems: 'center',
+				overflow: 'hidden',
+				fontSize: fontSizes[size],
+				'& a': {
+					display: 'flex',
+					alignItems: 'center',
+					'text-decoration': 'none',
+					color: theme.fn.themeColor(color, 6) +'!important'
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.styles.ts:43]: using `!important` is not a good idea, why this?

File: packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.svelte [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+	// retrieves the reactive context so that TimelineItem has access
+	// to the Timeline parameters
+	const state: BreadcrumbContext = getContext(ctx);
+
+	let separetor = $state.separetor;
+	size = $state.size;
+	color = $state.color;
+
+	$: ({ cx, classes, getStyles, theme } = useStyles({ color, size }, { name: 'Progress' }));
```
Comment (BeeMargarida) [inline packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.svelte:31]:
```suggestion
	$: ({ cx, classes, getStyles, theme } = useStyles({ color, size }, { name: 'BreadcumbItem' }));
```

**Review by BeeMargarida — COMMENTED** [review]
There's commits and changes for the radio component. This branch should only contain commits and changes of the breadcumb component.

Also, why the need for a BreadcumbItem component? Couldn't that logic be included inside the Breadcumb? It seems too much for a simple component

**Comment by masagatech** [comment]
> There's commits and changes for the radio component. This branch should only contain commits and changes of the breadcumb component.
> 
> 
> 
> Also, why the need for a BreadcumbItem component? Couldn't that logic be included inside the Breadcumb? It seems too much for a simple component

1. Oops is it let me check.
2. I think if there is ui tags for component it will have more control for users to manipulate the dom. It should be same design pattern isn't it?

**Pull request closed without merging**